numpy/add_newdocs.py

```diff
@@ -5463,18 +5442,22 @@ def luf(lamdaexpr, *args, **kwargs):
 
     Parameters
     ----------
-    X : array_like
-        First input array.
-    Y : array_like
-        Second input array.
-    out : array_like
-        An array to store the output. Must be the same shape as the
-        output would have.
+    *x : array_like
```

Comment:
> but I assume you would like it to eventually show up.

Not really - AFAICT, the sole purposed of this page is to redirect the user to `np.info` or the ufunc docs. if `ufunc.__doc__` is made visible in the web docs, then IMO, it should be the contents of `ufuncs.rst`.

This docstring aside - I was pretty sure that `**kwargs` was parsed correctly in the other docstring modified by this changeset - does the issue only exist for single asterisks?